src/main/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/transformer/cas2/StatusUpdateTransformer.kt

```diff
@@ -0,0 +1,26 @@
+package uk.gov.justice.digital.hmpps.approvedpremisesapi.transformer.cas2
+
+import org.springframework.stereotype.Component
+import uk.gov.justice.digital.hmpps.approvedpremisesapi.api.model.Cas2StatusUpdate
+import uk.gov.justice.digital.hmpps.approvedpremisesapi.jpa.entity.Cas2StatusUpdateEntity
+import uk.gov.justice.digital.hmpps.approvedpremisesapi.transformer.ExternalUserTransformer
+
+@Component("Cas2StatusUpdateTransformer")
+class StatusUpdateTransformer(
+  private val externalUserTransformer: ExternalUserTransformer,
+) {
+
+  fun transformJpaToApi(
+    jpa: Cas2StatusUpdateEntity,
+  ):
+    Cas2StatusUpdate {
+    return Cas2StatusUpdate(
+      id = jpa.id,
+      name = jpa.status().name,
```

Comment:
Is the reason `name` isn't a regular field, because while the `label` and `description` might change, the `name` will always stay the same, so we have to get that from `Cas2ApplicationStatusSeeding` directly?

Reply:
yes that's right -- the 'name' of the status comes the associated `Cas2ApplicationStatus`. At present these are just simple "model" or API objects hard-coded in `Cas2ApplicationStatusSeeding` but I guess we'll move them into the db and have the association managed by JPA with the `@ManyToOne` annotation, something like:

```kt
@ManyToOne
@JoinColumn(name = "status_id")
val status: Cas2ApplicationStatus,
```

 At that point the line in the transformer would become:

```kt
name = jpa.status.name,
```

I think...